cmake/lib/hello_world.h

```diff
@@ -0,0 +1,3 @@
+#include <iostream>
```

Comment:
You need to include those files which will be used in that file. So You are not using `iostream` here. Remove